src/DIRAC/WorkloadManagementSystem/Client/Matcher.py

```diff
@@ -224,14 +234,16 @@ def _processResourceDescription(self, resourceDescription):
 
     # Add 'MultiProcessor' to the list of tags
     if nProcessors and nProcessors > 1:
-      resourceDict.setdefault("Tag", []).append("MultiProcessor")
+      resourceDict.setdefault("Tag", []).extend("MultiProcessor")
```

Comment:
This will add MultiProcessor string character by character

Reply:
I mean:
    >>> ll = ["SomeTag"]
    >>> ll.extend("MultiProcessor")
    >>> ll
    ['SomeTag', 'M', 'u', 'l', 't', 'i', 'P', 'r', 'o', 'c', 'e', 's', 's', 'o', 'r']

Reply:
```suggestion
      resourceDict.setdefault("Tag", []).extend(["MultiProcessor"])
```